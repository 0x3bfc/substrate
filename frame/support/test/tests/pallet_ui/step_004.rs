#[frame_support::pallet]
mod pallet {
	#[pallet::trait_]
	struct Foo;
}

fn main() {
}
