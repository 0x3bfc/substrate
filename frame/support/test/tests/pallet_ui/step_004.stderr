error: Invalid pallet::config, expect trait definition
 --> $DIR/step_004.rs:4:2
  |
4 |     struct Foo;
  |     ^^^^^^
